google/cloud/security/iam/cli.py

```diff
@@ -0,0 +1,521 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" IAM Explain CLI. """
+
+#pylint: disable=too-many-locals,missing-type-doc,missing-param-doc
```

Comment:
You'll need to remove the doc ones as we are requiring proper Google style docs now.